bundle/edu.gemini.ags/src/main/scala/edu/gemini/ags/gems/mascot/util/MatrixUtil.scala

```diff
@@ -236,14 +236,14 @@ object MatrixUtil {
        }
     }
     sb.append(")\n")
-    sb.toString;
+    sb.toString
```

Comment:
I guess the original author hadn't discovered the `mkString` method.  As in `a.map(sFormat).mkString("Array(\n", ",\n", ")\n")`